Bylaws.md

```diff
@@ -112,15 +112,15 @@ This section applies to any ballot that proposes a Final Guideline or a Final Ma
     * The Table of Contents,
     * The year in the “Copyright” information,
     * Footers with page numbers,
-    * Links to other sections within the document.    
+    * Links to other sections within the document, and
+    * Converting relative effective dates (e.g. "6 months after publication") into absolute dates (e.g. "Month DD, YYYY").
```

Comment:
Effective dates are inherently normative, so I think the second-to-last sentence of the paragraph on line 110 needs to be amended to remove "informative (non-normative)".

(I can't comment/suggest changes directly on line 110 because Github won't let me).

Reply:
@aarongable would you also be ok to add

"markdown/formatting changes to tables, bulleted or numbered items"?

Reply:
Thanks! I've incorporated both of these. Please take a look and let me know what you think, and then I'll start a v2 discussion period.